apps/site/lib/site/schedule_note.ex

```diff
@@ -7,24 +7,37 @@ defmodule Site.ScheduleNote do
   import Phoenix.HTML.Link
   alias Phoenix.HTML
   alias SiteWeb.Router.Helpers
+  alias Schedules.HoursOfOperation
+  import SiteWeb.TimeHelpers, only: [format_time: 1]
 
   @type exception :: %{
           type: String.t(),
           service: String.t()
         }
 
-  defstruct peak_service: "", offpeak_service: "", exceptions: [], alternate_text: nil
+  @type hours_of_operation :: %{
+          first_departure: String.t(),
+          last_departure: String.t()
+        }
+
+  defstruct peak_service: "",
+            offpeak_service: "",
+            exceptions: [],
+            alternate_text: nil,
+            hours_of_operation: []
 
   @type t :: %__MODULE__{
           peak_service: String.t(),
           offpeak_service: String.t(),
           exceptions: [exception],
-          alternate_text: String.t() | nil
+          alternate_text: String.t() | nil,
+          hours_of_operation: [hours_of_operation]
```

Comment:
This type appears to be incorrect; `hours_of_operation` is set to the return value of the `hours` function, which is an `HoursOfOperation` struct. But, this struct doesn't align with its own typespec (`adjust_departures` changes the type of the fields). It looks like the purpose of "adjusting" the departures is to format the time values. Can we do this on the frontend? It seems like more of a view-layer concern.